src/main/java/com/amihaiemil/docker/RtDocker.java

```diff
@@ -158,6 +160,24 @@ public Plugins plugins() {
         );
     }
 
+    @Override
+    public Version version() throws IOException {
+        final String versionUri = this.baseUri.toString() + "/version";
+        final HttpGet version = new HttpGet(versionUri);
+        final JsonObject json = this.client.execute(
```

Comment:
@milux This variable is only used one time, please put it inline, in the call to ``new RtVersion()``, like this:

```java
return new RtVersion(
     this.client.execute(
            version,
            new ReadJsonObject(
                new MatchStatus(version.getURI(), HttpStatus.SC_OK)
            )
    ),
    this.client,
    URI.create(versionUri),
    this
);
```

Also, wrap everything in ``try/finally`` and release the connection, like [here](https://github.com/amihaiemil/docker-java-api/blob/master/src/main/java/com/amihaiemil/docker/RtDockerSystem.java#L74)

Reply:
Done, also improved structure according to the pattern I found in `Inspection.java`.